packages/flutter_markdown/pubspec.yaml

```
@@ -4,7 +4,7 @@ description: A Markdown renderer for Flutter. Create rich text output,
   formatted with simple Markdown tags.
 repository: https://github.com/flutter/packages/tree/main/packages/flutter_markdown
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+flutter_markdown%22
-version: 0.7.3+2
+version: 0.7.3+3
```

[Linux repo_checks](https://ci.chromium.org/ui/p/flutter/builders/try/Linux%20repo_checks/11948/overview) got mad… I think (at least one of) the issues can be fixed by changing the version number like so:

```suggestion
version: 0.7.4
```